src/vn/edu/rmit/kuri/output/Display.java

```diff
@@ -174,7 +174,14 @@ public static void chart(Summary summary, Metric metric) {
       valuePositionOnChart.add(position);
     }
 
-    spacing = (int) Math.floor((float) (horizontal - 1) / valueForDisplay.size());  // TODO: This is not accurate. Fix it.
+    //create spacing between data points on the chart
+    spacing = (int) Math.floor((float) (horizontal) / valueForDisplay.size());
```

Comment:
I think you should keep horizontal -1 because the chart already uses 1 column for the y axis.

Reply:
True.